### PR TITLE
CC-14562: Add categories to discount query parameters.

### PR DESCRIPTION
**Pull request by dmiseev**
## PR Description

- Epic: https://spryker.atlassian.net/browse/CC-14562

- Ticket for IG: https://spryker.atlassian.net/browse/CC-14583

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Category Management + promotions & discounts feature integration
+description: This guide provides step-by-step instructions on integrating Category Management + Promotions & Discounts feature into a Spryker-based project.
```
Comment (JacquieRooney) [inline docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md:3]:
```suggestion
description: This guide provides step-by-step instructions on integrating into a Spryker-based project the new feature: Addition of a "Category" parameter to calculation and conditions queries in discounts.
```

File: docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the plugins work correctly:
+
+1. [Create a discount](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-cart-rules.html) and define its condition as a query string with a *category* field.
+2. Add a product with the defined category to cart.
```
Comment (JacquieRooney) [inline docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md:71]:
```suggestion
2. Add a product assigned to the defined category to cart.
```

**Review by JacquieRooney — CHANGES_REQUESTED** [review]
Just a few changes requested around the phrasing. I can't comment on the correctness of the integration instructions, but generally this looks good.

File: docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+title: Category Management + promotions & discounts feature integration
+description: his guide provides step-by-step instructions on integrating into a Spryker-based project the new feature: Addition of a "Category" parameter to calculation and conditions queries in discounts.
```
Comment (DmitryLymarenko) [inline docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md:3]:
```suggestion
description: This guide provides step-by-step instructions on integrating into a Spryker-based project the new feature: Addition of a "Category" parameter to calculation and conditions queries in discounts.
```

File: docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+To start feature integration, overview, and install the necessary features:
+
+
+| NAME                   | VERSION |
+|------------------------| --- |
+| Promotions & Discounts | {{page.version}} |
+| Category Management    | {{page.version}} |
+| Spryker Core           | {{page.version}} |
+
+### 1) Set up behavior
```
Comment (DmitryLymarenko) [inline docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md:22]: transfers section is missing

Comment (DmitryLymarenko) [reply docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md:22]: the same for frontend section 
CSS was adjusted

**Review by DmitryLymarenko — DISMISSED** [review]

**Review by JacquieRooney — DISMISSED** [review]

File: docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | INTEGRATION GUIDE |
+| --- | --- | --- |
+| Promotions & Discounts | {{page.version}} | [Promotions & Discounts feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/promotions-and-discounts-feature-integration.html) |
+| Category Management | {{page.version}} | [Category Management feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/category-management-feature-integration.html) |
+| Spryker Core | {{page.version}} | [Spryker Сore feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/spryker-core-feature-integration.html) |
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md:22]: @dmiseev check the links I added to the feature integration guides and say if they are correct or I should replace them.
[Promotions & Discounts feature integration](https://docs.spryker.com/docs/scos/dev/feature-integration-guides/202108.0/promotions-and-discounts-feature-integration.html)
[Category Management feature integration](https://docs.spryker.com/docs/scos/dev/feature-integration-guides/202108.0/category-management-feature-integration.html)
[Spryker Сore feature integration](https://docs.spryker.com/docs/scos/dev/feature-integration-guides/202108.0/spryker-core-feature-integration.html)

Comment (dmiseev) [reply docs/scos/dev/feature-integration-guides/202200.0/category-management-promotions-and-discounts-feature-integration.md:22]: They are correct

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**